addons/hr_timesheet/controllers/portal.py

```diff
@@ -138,7 +145,7 @@ def get_timesheets():
             )] if timesheets else []
             return timesheets, grouped_timesheets
 
-        timesheets, grouped_timesheets = get_timesheets()
+        timesheets, grouped_timesheets = get_timesheets(domain)
```

Comment:
Why do you do that?

Reply:
Sorry, I did not know it would cause issues.
And what if we rename the variable in get_timesheets and so in the read_group calls?
`read_group_domain = AND([domain, [('id', 'in', timesheets.ids)]])`
Thanks 🙂